frontend/src/main/components/Jobs/JobsTable.js

```diff
@@ -18,7 +31,7 @@ export default function JobsTable({ jobs }) {
       Header: "Status",
       accessor: "status",
     },
-    PlaintextColumn("Log", (cell) => cell.row.original.log),
+    PlaintextColumn("Log", (cell) => truncateLog(cell.row.original.log)),
```

Comment:
I would suggest moving the truncation feature into PlainttextColumn via an optional parameter maxLines that, if set, will truncate the output after maxLines lines, along with putting some text *outside* the `<pre>` element (i.e. in a different font) that indicates that the log was truncated, and how many additional lines of output there are.

That will enable you to test it better, and be more visually understandable for the end user.

Reply:
But if you don't do that, then at least do this:

```js
export function truncateLines = (lines) => lines.slice(0, 10).join("\n") + `[${lines.length - 10) additional lines truncated`;
```

So that the output will be something like:

```
Data is not stale for [ANTH 20221]
Data is not stale for [ART  CS 20221]
Data is not stale for [ARTHI 20221]
Data is not stale for [ARTST 20221]
Data is not stale for [AS AM 20221]
Data is not stale for [ASTRO 20221]
Data is not stale for [BIOL CS 20221]
Data is not stale for [BL ST 20221]
Data is not stale for [C LIT 20221]
Data is not stale for [CH E 20221]
[184 additional lines truncated]
```

Reply:
#31 addresses this issue